src/array/cuda/csr2coo.cu

```diff
@@ -85,6 +146,26 @@ COOMatrix CSRToCOODataAsOrder(CSRMatrix csr) {
   // The row and column field have already been reordered according
   // to data, thus the data field will be deprecated.
   coo.data = aten::NullArray();
+  coo.row_sorted = false;
```

Comment:
Why `row_sorted` and `col_sorted` were set to false when calling CSRToCOODataAsOrder?